pyfar/constants/speed_of_sound.py

```diff
@@ -52,3 +52,114 @@ def speed_of_sound_simple(temperature):
     t_0 = pf.constants.absolute_zero_celsius
     c_ref = pf.constants.reference_speed_of_sound
     return c_ref*np.sqrt((temperature-t_0)/(t_ref-t_0))
+
+
+
+def speed_of_sound_cramer(
+        temperature, relative_humidity, atmospheric_pressure=None,
+        co2_ppm=314):
+    r"""
+    Calculate the speed of sound in air based on temperature, atmospheric
+    pressure, humidity and CO\ :sub:`2` concentration.
+
+    This implements Cramers method described in [#]_.
+    The uncertainty in the speed of sound is estimated to be
+    less than 300 parts per million.
```

Comment:
The default is 314 and the uncertainty is 300? This does not sound right to me. Could there be a mistake here?
Also: standard or extended uncertainty?